packages/react-composites/src/composites/ChatComposite/file-sharing/FileUpload.ts

```diff
@@ -96,41 +100,30 @@ export interface FileUploadManager {
 }
 
 /**
- * An internal interface used by the Chat Composite to drive the UI for file uploads.
- * @beta
+ * A wrapper object for a file that is being uploaded.
+ * Provides common functions for updating the upload progress, canceling an upload etc.
+ * @internal
```

Comment:
If this is just used in this package mark as `private`, if its exported from this package but not publically keep as @internal but the name should be `_FileUpload`